lms/envs/aws.py

```diff
@@ -1038,3 +1038,6 @@
 EDEOS_API_SECRET = AUTH_TOKENS.get('EDEOS_API_SECRET', '')
 EDEOS_API_URL = AUTH_TOKENS.get('EDEOS_API_URL', '')
 # EDEOS
+
+# Number of courses per page setting (for home page)
+HOMEPAGE_COURSE_MAX = ENV_TOKENS.get('HOMEPAGE_COURSE_MAX', HOMEPAGE_COURSE_MAX)
```

Comment:
This change is not related to the issue.

Reply:
fixed